test-automation-school/src/test/java/test/automation/school/HWTest.java

```diff
@@ -0,0 +1,87 @@
+package test.automation.school;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import test.automation.school.pageobjects.*;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Created by Max on 5/17/2017.
+ */
+public class HWTest extends BaseTest {
+    HomePage homePage;
+    SearchPage searchPage;
+    SearchResultsPage searchResultsPage;
+
+    @Test
+    public void ssLvSearchTest() {
+        // 1. Открыть браузер и развернуть на весь экран.
+        // 2. Зайти на ss.lv.
+        homePage = new HomePage(driver);
+
+        // 3. Поменять язык на русский.
+        homePage.switchLangTo("RU");
+
+        //3a. navigate to Search page
+        homePage.pageHeaderSearch.click();
+        searchPage = new SearchPage(driver);
+
+        // 4. в поиске ввести искомую фразу (напр. Компьютер....)
+        searchPage.lookupTxt.sendKeys("Компьютер");
+        //searchPage.lookupTxt.sendKeys("Компьютеры, оргтехника");
+
+        // 5. выбрать разные параметры поиска
```

Comment:
Every each dropdown in this case should have separate method to interact with it. Test should not "know" about the "internals" of the page and elements. What if the implementation will change - instead of a dropdown, there will be a combo box or etc. Then you will have to change both - the tests and the page objects. Using "interface" approach, you will change only Page Object layer.